product_docs/docs/pem/9/monitoring_performance/probes.mdx

```diff
@@ -166,6 +166,43 @@ Use the **General** tab to modify the definition of an existing probe or to spec
     To invoke a script on a Windows system, set the registry entry for `AllowBatchJobSteps` to `true` and restart the PEM agent.
 
 -   Use the **Target Type** list to select the object type for the probe to monitor. **Target Type** is disabled if **Collection method** is WMI.
+    probes are used to collect data from different levels of the system hierarchy. The Target Type of a probe defines the specific level within the system hierarchy where the probe will collect data. This ensures that the probe     
+    gathers relevant metrics from the appropriate components of the system.
+
+    <details><summary>Target Type Configuration Matrix</summary>
+    | Target Type | Execution Level | Columns That the Probe Query Must return              | Probe examples |
+    |-------------|-----------------|-------------------------------------------------------|----------------|
+    | Agent       | Agent           | None                                                  | cpu_usage      |
+    | Server      | Server          | None                                                  |                |
+    | Database    | Database        | None                                                  |                |
+    | Schema      | Database        | schema_name                                           |                |
+    | Table       | Database        | schema_name, table_name                               |                |
+    | Index       | Database        | schema_name, index_name                               | index_size     |
+    | Sequence    | Database        | schema_name, sequence_name                            |                |
+    | View        | Database        | schema_name, view_name                                |                |
+    | Function    | Database        | schema_name,  arg_types, function_type, function_name |                |
+    | Extension   | Extension       | None                                                  | Extension      |
+
+    **Column Descriptions**
+    Target Type:
+    Specifies the target object that the probe will monitor. Each target type corresponds to a different kind of database object or level.
+
+    Execution Level:
+    Indicates the frequency at which the probe Executes Once Per object. This could be once per server, database, or other specified levels, ensuring that the monitoring is appropriately scoped.
+    Note that the Custom probe set to a target type of Database or higher (including Schema, Table, Index, View, Sequence, Functions) will be displayed at the database level and System level probes set to a target type of Schema or 
+    higher will be displayed until the schema level.
+
+    Columns That the Probe Query Must return:
```

Comment:
```suggestion
    Probe query columns:
```

Reply:
same as above...need to come up with more appropriate name